charts/infra-apps/templates/NOTES.txt

```diff
@@ -0,0 +1,13 @@
+The following Namespaces were configured:
+{{- range $name, $data := .Values.namespaces.namespaces }}
```

Comment:
Maybe namespaces should be created by an argo event or workflow.

Reply:
I removed the Namespace issue from the scope of this chart for now. I believe that we should manage namespaces (and maybe also rbac rules) somewhere else so we can isolate security concerns related to namespaces from the actual app deployments.

We will also want to use whatever solution we choose to manage namespaces for more than just infra apps and we don't want to have to reconfigure infra-apps when we need to a namespace for something. Maybe namespaces should be treated as data and not as code... I need to think about this a bit more and will gladly take pointers to ideas that help us keep namespaces somewhat dynamic while still gitopsing them..